zio-k8s-operator/src/main/scala/com/coralogix/zio/k8s/operator/leader/package.scala

```diff
@@ -19,28 +19,28 @@ package object leader {
       /** Runs the given effect by applying the leader election algorithm, with the guarantee that
         * the inner effect will only run at once in the Kubernetes cluster.
         *
-        * If you want to manage the lock as a ZManaged use [[lease]]
+        * If you want to manage the lock as Scoped use [[lease]]
         *
         * @param f
         *   Inner effect to protect
         */
-      def runAsLeader[R, E, A](f: ZIO[R, E, A]): ZIO[R with Clock, E, Option[A]] =
+      def runAsLeader[R, E, A](f: ZIO[R, E, A]): ZIO[R with Scope, E, Option[A]] =
```

Comment:
Same thing here about extending leaking out the scope. This should be a `ZIO` not requiring a `Scope`, and a `ZIO.scoped` used around the `lease` inside.

Reply:
done.